benchmark/pytorch_tabular_benchmark.py

```diff
@@ -30,7 +30,7 @@
 parser.add_argument('--batch_size', type=int, default=256)
 parser.add_argument('--epochs', type=int, default=1)
 parser.add_argument('--model_type', type=str, default='TabTransformer',
-                    choices=['TabTransformer'])
+                    choices=['TabTransformer', 'FTTransformer'])
```

Comment:
Is it possible to include TabNet?

Reply:
The tab net implementation in tabular is super different between pyg and frame. I can't get the models to have approximate the same number of parameters.